src/VM/Handler/Opcode8138Handler.cpp

```diff
@@ -40,9 +42,40 @@ namespace Falltergeist
 
             void Opcode8138::_run()
             {
+                //TODO: search ContainerItemObject and CritterObject inventories recursively if it has container itemobjects in it
+                //for now recursive inventories are not yet implemented since ItemObject has no inventory vector
+                
                 Logger::debug("SCRIPT") << "[8138] [=] int item_caps_total(void* obj)" << std::endl;
-                _script->dataStack()->popObject();
-                _script->dataStack()->push(0);
+                auto object = _script->dataStack()->popObject();
+                if (auto container = dynamic_cast<Game::ContainerItemObject*>(object))
+                {
+                    for(auto item: *container->inventory())
+                    {   
+                        if(item->PID() == 519)
```

Comment:
It's bad practice to hard-code magic numbers. Replace with ENUM reference (as temporary solution), or implement properly (like checking if it's an actual container).